hera_pspec/pspecbeam.py

```diff
@@ -429,7 +440,7 @@ def power_beam_sq_int(self, pol='pI'):
         primary_beam_area: float, array-like
         """
         if hasattr(self.primary_beam, 'get_beam_area'):
-            return self.primary_beam.get_beam_sq_area(pol)
+            return np.real(self.primary_beam.get_beam_sq_area(pol))
```

Comment:
Should this function be returning complex numbers to begin with? Or is there a bug in `get_beam_sq_area`?

Reply:
That's what I was asking @bhazelton, and I never got a response, but I think I came to the conclusion that in principle it can be complex (e.g. when asking for `XY` and `YX` areas), but in practice we will always be using `XX` or `YY` or `pI` or `pQ` beams, in which case it is real. But @bhazelton let me know if you have any opinions on that.